cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax-alter-user.asciidoc

```diff
@@ -1,12 +1,16 @@
 .Command syntax
-[source, cypher]
+[source]
```

Comment:
Why remove `cypher`? We have those everywhere else in the code examples. If there is a reason for removing it, that's fine, but it should be consistent.

Reply:
These are not cypher examples, they're syntax description. Since this causes the docs to generate an "Execute in browser" block, anything marked in this way should actually be executable. Agree this should be consistent though, we've put a card on our wall to clean it up in the other places.

Reply:
I see and in that case, I agree with you, let's remove them here.